vuepress/docs/next/tutorials/devops/manage-nginx.md

```diff
@@ -0,0 +1,175 @@
+---
+sidebarDepth: 2
+---
+# Manage NGINX
+
+## Verify the NGINX Ingress Install
+We recommend setting up a test application to verify that the ingress is working.
+
+1. Create a simple application from the `Cloud Shell`:
+```
+kubectl create deployment hello-server --image=us-docker.pkg.dev/google-samples/containers/gke/hello-app:1.0
+```
+
+2. Expose the `hello-app` deployment as a Service:
+```
+kubectl expose deployment hello-server --type LoadBalancer --port 80 --target-port 8080
+```
+
+3. Create this `ingress-resource.yaml` file:
+```
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingress-resource
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+spec:
+  rules:
+  - http:
+      paths:
+      - path: /hello
+        pathType: Prefix
+        backend:
+          service:
+            name: hello-server
+            port: 
+              number: 80
+```
+4. Create the Ingress Resource:
+```
+kubectl apply -f ingress-resource.yaml
+```
+
+5. Verify that the Ingress Resource has been created:
+```
+kubectl get ingress ingress-resource
+```
+It may take several minutes to populate the `Address`.
+
+6. Verify access to the web application using the `EXTERNAL-IP/hello` address of the previous `nginx-ingress-controller`. You should see the following:
+```
+Hello, world!
+Version: 1.0.0
+Hostname: hello-app
+```
+Note that you will need the EXTERNAL-IP address of your ingress controller to configure the application.
+
+7. Set `ENTANDO_INGRESS_CLASS` to `nginx`. Entando exposes this environment variable to establish the ingress controller used for the deployment. 
```

Comment:
This is a bit confusing and I think out-of-date. 
"Verify that you configured the ingress class in the Operator ConfigMap:  `entando.ingress.class: "nginx"` so Entando knows which ingress controller should be used."